routes.json

```diff
@@ -1,9 +1,3 @@
 {
```

Comment:
I tried the same early on during the development but it wasn't loading the other pages if I refreshed the browser. Say I'm on `/create` page, now if I reload the browser, it'll give a `404` not found.

This will become problematic when users start sharing links to their blogs, ex - `https://myblog.point/blog?id=some-id`. They will all result in `404`s.